docs/cookbook/recipe_custom_view.rst

```diff
@@ -0,0 +1,134 @@
+Creating a Custom Admin View
+============================
+
+Sometimes you may want to create an additional view in your SonataAdmin
+but you do not have or need an Entity to connect it to. You also probably
+want to add it to sidebar with the rest of your admin pages? Here we will
+present a full working example of how is that achievable.
```

Comment:
- "Sometimes,"
- "Sonata admin"
- "to the sidebar"
- "admin pages."
- "how that is achievable"